NEWS.md

```diff
@@ -27,6 +27,7 @@
 
 ## 2020-07-26 (0.9.13)
 
+* `company-select-mouse` is now a frontend action.
```

Comment:
That line was a bit misplaced, though.

Reply:
Where should it go instead? I can make a PR to correct this.

Reply:
Already fixed, see the latest `master`.